docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -18,6 +18,19 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## No basic role
+
+<!-- Eric Leijonmarck -->
+<!-- OSS, Enterprise -->
+
```

Comment:
Hey @eleijonmarck , can you please add a date for this entry on (or close to) this line? Format should be just regular text (not bold or a heading) like this: September 29, 2023